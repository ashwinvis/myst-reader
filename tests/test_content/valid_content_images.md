---
title: Valid Content with Image
author: My Author
Date: 2020-10-16
---
This is file contains a image.

![Image alt title](/path/to/image.png)

[![Image with link](/path/to/link.png)](https://example.com/link.png)

```{image} {filename}/path/to/image.png
:alt: using image directive with backticks
```

:::{image} {filename}/path/to/image.png
:alt: using image directive with colon fence
:::
