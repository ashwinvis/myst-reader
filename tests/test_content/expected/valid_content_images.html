<p>This is file contains a image.</p>
<p><img src="/path/to/image.png" alt="Image alt title" /></p>
<p><a href="https://example.com/link.png"><img src="/path/to/link.png" alt="Image with link" /></a></p>
<img src="{filename}/path/to/image.png" alt="using image directive with backticks"/>
<img src="{filename}/path/to/image.png" alt="using image directive with colon fence"/>